### DISCUSSION TITLE
Maps - how to integrate side-by-side (before-after) comparer?

### DISCUSSION THREAD

**Discussion by lukesdm** [quarto-dev/quarto-cli#845] [Q&A] [answered]
Hi, I'm wondering what the best way to integrate something like [this](https://github.com/digidem/leaflet-side-by-side) - a side-by-side/before-after map slider - would be?

![maps](https://github.com/digidem/leaflet-side-by-side/blob/gh-pages/screencast.gif?raw=true)

I guess there are 2 questions: 1. how to integrate a leaflet plugin like that, and 2) how to integrate something similar for other widget types e.g. images?

**Comment by jimjam-slam** [comment, marked as answer]
For images I'd check out Knight Lab's [JuxtaposeJS](https://github.com/NUKnightLab/juxtapose). Here's a demo Quarto doc that uses it!

Source: https://github.com/jimjam-slam/quarto-experiments/blob/main/juxtapose/index.qmd
Rendered: https://jimjam-slam.github.io/quarto-experiments/juxtapose

**Reply by lukesdm** [reply]
> For images I'd check out Knight Lab's [JuxtaposeJS](https://github.com/NUKnightLab/juxtapose). Here's a demo Quarto doc that uses it!
> 
> Source: https://github.com/jimjam-slam/quarto-experiments/blob/main/juxtapose/index.qmd Rendered: https://jimjam-slam.github.io/quarto-experiments/juxtapose

Fantastic, thank you!
